build-email.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
```

Comment:
Is this a file generated every time? If so, these files should be in gitignore.

Reply:
I don't know, it would be better to ask Justin. I only know how to use the methods Justin gave me.